### ISSUE TITLE
Label comments and add a section to introduce the labels

### ISSUE THREAD

**Issue by Bai-Li-NOAA** [NOAA-FIMS/collaborative_workflow#107]
**Is your feature request related to a problem? Please describe.**
We discussed how to provide feedback during code review in one of the FIMS project meetings. Labeling comments during code review could potentially save hours of communications and misunderstandings. Comments with labels are parseable by machines. They are also helpful for labeling the severity of the comments, differentiating required changes from suggestions (see [Google Engineering Practices Documentation](https://google.github.io/eng-practices/review/reviewer/comments.html#label-comment-severity)). 

**Describe the solution you'd like**
- Encourage reviewers to use all or some of the labels from [conventionalcomments.org](https://conventionalcomments.org/). [Edited] For example, praise, ~~nitpick~~ quibble, suggestion with _blocking_ or _non-blocking_ **decorations**, question, and thought.
- Use decorations to give additional context for a comment and help authors to prioritize comments/tasks. For example, A comment with _non-blocking_ **decoration** should not prevent the subject under review from being accepted. A comment with _blocking_ **decoration** should prevent the subject under review from being accepted, until it is resolved.
- Format of a comment
```
<label> [decorations]: <subject>
[discussion]
```
```
- label - This is a single label that signifies what kind of comment is being left.
- subject - This is the main message of the comment.
- decorations (optional) - These are extra decorating labels for the comment. They are surrounded by parentheses and comma-separated.
- discussion (optional) - This contains supporting statements, context, reasoning, and anything else to help communicate the “why” and “next steps” for resolving the comment.
```
Document labels, decorations and format in the collaboration workflow book.

**Describe alternatives you've considered**
Too much work :sleepy: Let's not label comments.

**Additional context**
NA

**Comment by ChristineStawitz-NOAA** [comment]
I LOVE this idea @Bai-Li-NOAA thanks for suggesting! We followed it in our code review session today.

I did have a kid with lice last week so nitpicking brings up bad memories. @iantaylor-NOAA suggested maybe we call it "quibble" instead? I may also be fine with the term nitpicking once I'm sure all the lice are gone.

**Comment by msupernaw** [comment]
Just went through the whole lice thing a couple of weeks ago! Sent from my iPhoneOn Jun 9, 2023, at 6:43 PM, Christine Stawitz - NOAA ***@***.***> wrote:﻿
I LOVE this idea @Bai-Li-NOAA thanks for suggesting! We followed it in our code review session today.
I did have a kid with lice last week so nitpicking brings up bad memories. @iantaylor-NOAA suggested maybe we call it "quibble" instead? I may also be fine with the term nitpicking once I'm sure all the lice are gone.
